frontend/src/app/Settings/Nav.tsx

```diff
@@ -1,13 +1,17 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
 
+const getFacilityIdFromUrl = () => {
```

Comment:
I get an error when i try to import this from `../utils/url`, but it works fine from index. I think its looking in node_modules